packages/react/tests/server.test.tsx

```diff
@@ -11,6 +15,11 @@ describe('environment', () => {
   });
 });
 
+// Test for #1558
```

Comment:
We have this here for react and also for ember tests, but no tests have been updated for the other frameworks. Is there any reason for that?

I would prefer to have them consistent. But also I wonder if we really need to do this everywhere? Would the unit test you added [here](https://github.com/simonihmig/responsive-image/pull/1560/files#diff-27cee72a7f66a31f83560d1567f34f6611197b5ca8383da93d6a1d32d9328c7bR43) not be sufficient to cover this feature everywhere? 🤔

Reply:
I mostly added them to sanity check and confirm the end result was as expected 😅 I can remove them to keep things consistent.